packages/core/core/src/Transformation.js

```diff
@@ -158,6 +162,14 @@ export default class Transformation {
           },
         ]);
       }
+
+      if (existing == null) {
+        // If no existing sourcemap was found, initialize sourcesContent with
+        // the asset's filepath and its original contents.
+        asset.sourcesContent = {
+          [asset.value.filePath]: await asset.getCode(),
```

Comment:
This has to definitely be a relative path of some sorts

Reply:
The other sourcescontents are relative to projectroot as enforced by the sourcemap library